Mage/src/main/java/mage/abilities/effects/common/LearnEffect.java

```diff
@@ -22,7 +22,7 @@
 
     public LearnEffect() {
         super(Outcome.Neutral);
-        staticText = "learn. <i>(You may reveal a Lesson card you own from outside the game " +
+        staticText = "Learn. <i>(You may reveal a Lesson card you own from outside the game " +
```

Comment:
this needs to stay lower case, don't worry about it leading to incorrect text (I'm gonna do a big text overhaul once the set is fully implemented)

Reply:
fixed